IFTPipeline.jl/src/cli.jl

```diff
@@ -51,6 +51,192 @@ function mkclipreprocess_single!(settings)
         "--output", "-o"
         help = "Path to output segmented image file (.tiff)"
         required = true
+        
+    end
+    return nothing
+end
+
+function mkcli_preprocess_tiling_single!(settings)
+    @add_arg_table! settings["preprocess_tiling_single"] begin
+        "--truecolor", "-t"
+        help = "Truecolor image file (.tiff)"
+        required = true
+
+        "--falsecolor", "-r"
+        help = "Falsecolor image file (.tiff)"
+        required = true
+
+        "--landmask-dilated", "-d"
+        help = "Landmask image file (dilated, .tiff)"
+        required = true
+
+        "--segmented", "-s"
+        help = "Path to output segmented image file (.tiff)"
+        required = true
+        
+        "--labeled", "-l"
+        help = "Path to output labeled image file (.tiff)"
+        required = true
+
+
+        # Tiling parameters
+        
+        "--tile_rblocks"
```

Comment:
Use `-` as separator for cli options for consistency.
```suggestion
        "--tile-rblocks"
```

Reply:
Yes, this needs to be checked for consistency with the parameter names in the cylc file too.